kernel/kselftest.py

```diff
@@ -153,6 +156,49 @@ def test(self):
         if self.error:
             self.fail("Testcase failed during selftests")
 
+    def kselftestengine(self, arg_payload):
+        cmd_output = subprocess.Popen(arg_payload, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+        output, error = cmd_output.communicate()
+        output_payload = output.decode("utf-8")
+        self.log.info("=============./ksm_tests Logs============")
+        self.log.info(output_payload)
+
+    def ksmtest(self):
+        self.test_type = self.params.get('test_type', default='-H')
+        self.Size_flag = self.params.get('Size', default='-s')
+        self.Dup_MM_Area = self.params.get('Dup_MM_Area', default='100')
+        ksm_test_dir = self.sourcedir + "/" + "vm" + "/"
+        if os.path.exists(ksm_test_dir):
+            os.chdir(ksm_test_dir)
+            if(self.test_type == "-H" or self.test_type == "-P"):
+                arg_payload = [r"./ksm_tests", self.test_type, self.Size_flag, self.Dup_MM_Area]
```

Comment:
please make sure you are following the proper indentation like length > 79 char.
run pycodestyle and pylint before pushing the code.